custom-routes-vis/src/main/java/com/google/sps/servlets/GetDirectionsServlet.java

```diff
@@ -59,48 +67,54 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /** 
-   * Sends request to the Directions API and returns the connection.
+   * Sends GET request to the Directions API and returns the connection.
    * @param origin Origin of the requested routes in "lat,lng" format.
    * @param destination Destination of the requested routes in "lat,lng" format.
-   * @param key The API key.
+   * @param apiKey The API key.
    */
-  public URLConnection sendGetRequestToDirectionsApi(String origin, String destination, String key) 
+  public URLConnection connectToDirectionsApi(String origin, String destination, String apiKey) 
       throws IOException, MalformedURLException {
-    String originDestination = "&origin=" + origin + "&destination=" + destination;
     URL directionsUrl = new URL("https://maps.googleapis.com/maps/api/directions/json?"
-          + "mode=driving"
-          + "&alternatives=true" + originDestination + "&key=" + key);
+        + "origin=" + origin + "&destination=" + destination    
+        + "&mode=driving"
+        + "&alternatives=true" + "&key=" + apiKey);
```

Comment:
You formatting here seems strange. Either give each  key value pair its own line, or fill out the 100 characters. i.e. Move the alternatives value up onto line 79.

Reply:
ok